From: Victor Seva <linuxmaniac@torreviejawireless.org>
Date: Tue, 22 Oct 2019 10:41:16 +0200
Subject: no_tls_openssl_mutex_shared_util

>
https://github.com/kamailio/kamailio/commit/a0ca4cd43f098d10aa5aadcc18453cbd0120dcef

```
**IMPORTANT: the workaround of using this preloaded shared library is no longer
needed starting with Kamailio v5.3.0-pre1 (git master branch after September 14, 2019).
The code of this shared library has been included in the core of Kamailio and the
same behaviour is now achieved by default.**
```
---
 src/modules/tls/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/tls/Makefile b/src/modules/tls/Makefile
index 224bbd8..3ea90a9 100644
--- a/src/modules/tls/Makefile
+++ b/src/modules/tls/Makefile
@@ -34,7 +34,7 @@ LIBS+= $(TLS_EXTRA_LIBS)
 # dcm: tls.cfg installed via local 'install-cfg' to update paths
 #MOD_INSTALL_CFGS=tls.cfg
 
-MOD_INSTALL_UTILS=utils/openssl_mutex_shared
+MOD_INSTALL_UTILS=
 
 include ../../Makefile.modules
 
